briana_bennett/Flask/greatNumberGame/server.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, render_template, request, redirect, session, flash
+app = Flask(__name__)
+app.secret_key = "ImASecret"
+import random
+
+@app.route('/')
+def index():
+	if 'randomNumber' not in session:
+		session['randomNumber'] = random.randint(1, 100)
+	
+	print "got random"
+	print session
```

Comment:
The following if statements could be condensed. `if "low_high" in session or "guess" in session:`